gcore/src/lib.rs

```diff
@@ -67,7 +67,7 @@ pub mod errors;
 pub mod exec;
 pub mod msg;
 pub mod prog;
-pub mod stack_buffer;
+pub use gstack_buffer as stack_buffer;
```

Comment:
pub extern crate

Reply:
This is bad for documentation.

`pub use gstack_buffer as stack_buffer;`
![image](https://github.com/gear-tech/gear/assets/109800286/fed8e602-32b9-4a52-8321-edfcdc8de3eb)
`pub extern crate gstack_buffer as stack_buffer;`
![image](https://github.com/gear-tech/gear/assets/109800286/2388fa2c-628f-456f-8a8d-6c935db58684)